python/lsst/obs/base/cameraMapper.py

```diff
@@ -1259,27 +1316,14 @@ def exposureFromImage(image, dataId=None, mapper=None, logger=None, setVisitInfo
     elif isinstance(image, afwImage.DecoratedImage):
         exposure = afwImage.makeExposure(afwImage.makeMaskedImage(image.getImage()))
         metadata = image.getMetadata()
-        try:
```

Comment:
It's not clear to me why this try/except block only existed in the case of a DecoratedImage before (but not for the case of a MaskedImage or Image). Has something about DecoratedImages and WCSs changed?

Reply:
I have no idea why it was originally there. But we're making the SkyWcs in an entirely separate place, so this block is now unnecessary.